book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(_title, author)
+    @title = tittle
+    @author = author
+    @rentals = []
+  end
+
+  def add_rentals(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- Hey, please, let's be able to keep track of the date and person renting a particular book as this would enforce the relationship a book has with a person since a person can have many books and a book can belong to a person, so thereby cementing the relationship between the rental. Here's the way to achieve that below:

```ruby

def add_rental(person, date)
    Rental.new(date, person, self)
  end
```